node_common/constants.js

```diff
@@ -1,5 +1,6 @@
 export const POLLING_RATE = 5000;
 export const POWERGATE_HOST = "https://grpcweb.slate.textile.io";
+export const IPFS_GATEWAY_DOMAIN = "ipfs.slate.textile.io";
```

Comment:
domain here, not 'url' since it needs to be prepended by cid still and protocol (https) added to be used